backend/tests/controller/GenerateReportController.test.ts

```diff
@@ -1,5 +1,7 @@
 import "mocha";
+// @ts-ignore
 import chai, { expect } from "chai";
+// @ts-ignore
```

Comment:
same

Reply:
My IDEA shows the alerts below, and I don't want to see the annoying red underline so I just ignored them.

- TS1259: Module '"chai"' can only be default-imported using the 'esModuleInterop' flag 
- TS1259: Module '"/Users/aaron/code/au/HeartBeat/backend/node_modules/chai-http/types/index"' can only be default-imported using the 'esModuleInterop' flag

Reply:
> My IDEA shows the alerts below, and I don't want to see the annoying red underline so I just ignored them.
> 
> * TS1259: Module '"chai"' can only be default-imported using the 'esModuleInterop' flag
> * TS1259: Module '"/Users/aaron/code/au/HeartBeat/backend/node_modules/chai-http/types/index"' can only be default-imported using the 'esModuleInterop' flag

could you please fix that issue by code, I think it should be fixed by tsconfig
try to make allowSyntheticDefaultImports: true, if it does not work, could you please do some investigation

Reply:
Yeah, I have fixed this issue in the latest commit by code without ignoring.